src/components/compress/result-cache.ts

```diff
@@ -15,7 +14,7 @@ interface CacheResult {
 interface CacheEntry extends CacheResult {
   preprocessorState: PreprocessorState;
   encoderState: EncoderState;
-  source: SourceImage;
+  sourceData: ImageData;
```

Comment:
The source image now has data that doesn't matter to the result cache (eg, rotation settings).